www/docs/en/dev/config_ref/index.md

```diff
@@ -320,8 +330,8 @@ platform. See [Icons and Splash Screens](images.html) for more information.
 
    Attributes(type) | Description
    ----------------- | ------------
-   name(string) | *Required* <br/> Allowed values: android-package, ios-package, onload. <br/> ==iOS== ==Android== <br/> 'ios-package' and 'android-package' are used to specify the name of the package (as specified by the 'value' attribute) to be used to initialize the plugin code, while 'onload' is used to specify whether the corresponding plugin (as specified in the 'value' attribute) is to be instantiated when the controller is initialized.
-   value(string or boolean) | *Required* <br/> ==iOS== ==Android== <br/> Specifies the name of the package to be used to initialize the plugin code (when the 'name' attribute is android-package or ios-package), specifies the name of the plugin to be loaded during controller initialization (when 'name' attribute is set to 'onload').
+   name(string) | *Required* <br/> Allowed values: android-package, ios-package, onload. <br/> ==iOS== ==OS X== ==Android== <br/> 'ios-package' and 'android-package' are used to specify the name of the package (as specified by the 'value' attribute) to be used to initialize the plugin code, while 'onload' is used to specify whether the corresponding plugin (as specified in the 'value' attribute) is to be instantiated when the controller is initialized.
```

Comment:
thinking of this...maybe we should be more strict. and only allow `osx-package` in the future. created CB-10602